### DISCUSSION TITLE
[Q/A] Using space bar Doesn't Pause Music

### DISCUSSION THREAD

**Discussion by Noob-linux-user** [KRTirtho/spotube#226] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

Pressing Space bar doesn't Pause Music

### Expected Behavior

Using space bar should pause music

### Spotube Version

v2.4.1

### Operating System

Linux

### Linux Distribution (if applicable)

.Deb package

### Operating System Version

Linux Mint 20.3 (based on ubuntu lts)

### Installation Source

Github Release (Binary)

**Comment by KRTirtho** [comment]
It's not a bug. It was never implemented
